crates/pop-cli/src/commands/build/contract.rs

```diff
@@ -16,9 +14,6 @@ pub struct BuildContractCommand {
 	/// usage. For production, always build in release mode to exclude debug features.
 	#[clap(short, long)]
```

Comment:
Do we still need those here as it is not a command anymore. That also gives reason to change the name of the type.

Reply:
The above `path` field can also already be initialised and thus remove the `Option`

Reply:
No need for macros anymore, cleaned in https://github.com/r0gue-io/pop-cli/pull/394/commits/b0b5946973c0f24ff07ef0d539ecd381b33f8fc4

For contracts is ok to keep path as Optional as the `build_smart_contract` method in the `pop_contracts` crate is expecting an Optional value.